src/pykeen/nn/init.py

```diff
@@ -109,3 +111,27 @@ def init_quaternions(
     imag = imag * (modulus * phase.sin()).unsqueeze(dim=-1)
     x = torch.cat([real, imag], dim=-1)
     return x.view(num_elements, 4 * dim)
+
+
+def create_init_from_pretrained(
+    pretrained: torch.FloatTensor,
+) -> Initializer:
+    """
+    Create an initializer via a constant vector.
+
+    :param pretrained:
+        the tensor of pretrained embeddings.
+
+    :return:
+        an initializer, which fills a tensor with the given weights.
+    """
+
+    def init_from_pretrained(
+        x: torch.FloatTensor,
+    ) -> torch.FloatTensor:
+        """Initialize tensor with pretrained weights."""
+        if x.shape != pretrained.shape:
+            raise ValueError(f"shape of pretrained {pretrained.shape} does not match shape of tensor {x.shape}")
+        return pretrained
```

Comment:
shouldn't this modify x in place?

Reply:
no, it does not need to. 

We decided against it to have an easier time defining composed initializations (without having to define an inplace `normalize` etc.)